### PR TITLE
#3 rename the package name from ndart to ndart_cli

### PR DESCRIPTION
**Pull request by IgorBabalich**
- Rename the package name from ndart to ndart_cli in order to free up this package name
  for the main angular package
- small dependency packages version updates
- set // @dart=2.9 in main.dart to prevent null sound issue messages until ngcomponents will be updated
- added build_version to update project version in the package sources
- .gitattributes to keep LF permanent line endings in the project to prevent different git configs

### CODE REVIEW

**Review by GZGavinZhao — CHANGES_REQUESTED** [review]
There's no need to opt-out of null safety. The current published version of `ngcomponents` works fine with null safe code.

**Comment by GZGavinZhao** [comment]
Looks good to me at first glance! I'll do some thorough testing tomorrow.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [1.2.0] - 2022-06-03
+
+### Changed 
+- Rename the package name from ndart to ndart_cli in order to free up this package name 
+  for the main angular package
+- small dependency packages version updates
+- set // @dart=2.9 in main.dart to prevent null sound issue messages until ngcomponent will be updated
```
Comment (GZGavinZhao) [inline CHANGELOG.md]: Remove this line, capitalize the first letter of every sentence, and then we're good to go :D

**Review by GZGavinZhao — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (IgorBabalich) [reply CHANGELOG.md]: ok, updated

**Comment by GZGavinZhao** [comment]
Looks good to me. Thanks!

**Pull request merged**